app/_includes/sidebar_link.html

```diff
@@ -1,15 +1,18 @@
 {% assign item = include.item %}
 {% assign current_page =  page.url | remove: '.html' %}
 
-<!-- "#TODO: handle external links" -->
-{% assign url = '/docs/' | append: include.version | append: item.url %}
+{% if item.absolute_url %}{% assign url = item.url %}
+{% else %}{% assign url = '/docs/' | append: include.version | append: item.url %}
+{% endif %}
 <a href="{{ url }}" class="sidebar-link {% if current_page == url %} active {% endif %}">{{ item.text or item.title }}</a>
 
 {% if item.items and item.items.size > 0 %}
   <ul class="sidebar-sub-headers {% if current_page != url %} hidden {% endif %}">
     {% for sub_item in item.items %}
       <li class="sidebar-sub-header">
-        {% assign url = '/docs/' | append: include.version | append: sub_item.url %}
+        {% if sub_item.absolute_url %}{% assign url = sub_item.url %}
```

Comment:
Would it make sense to just have `sub_item.absolute_url` just be the value or use the fact that url starts with a scheme?

Reply:
This implementation matches how docs.konghq.com does it